src/NzbDrone.Api/Authentication/EnableAuthInNancy.cs

```diff
@@ -31,5 +53,40 @@ private Response RequiresAuthentication(NancyContext context)
 
             return response;
         }
+
+        private void RegisterFormsAuth(IPipelines pipelines)
+        {
+            if (_configFileProvider.AuthenticationMethod != AuthenticationType.Forms)
+            {
+                return;
+            }
+
+            var salt = new byte[] {1, 2, 3, 4, 5, 6, 7, 8};
```

Comment:
Should we treat the salt the same way we deal with the  Passphrases, where we create it once and store it in the db?